package.json

```
@@ -28,10 +28,13 @@
   "keywords": [],
   "license": "MIT",
   "devDependencies": {
+    "@types/jest": "^26.0.23",
```

```suggestion
    "@types/jest": "26.0.23",
```

We pin all the versions we use in our `package.json`. 😉